support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -344,39 +344,36 @@ export const productCatalogDescription: Record<ProductKey, ProductDescription> =
 		},
 	};
 
-export function productCatalogDescriptionNewBenefits(
-	countryGroupId: CountryGroupId,
-): Record<ProductKey, ProductDescription> {
-	return {
-		...productCatalogDescription,
-		TierThree: {
-			...productCatalogDescription.TierThree,
-			benefits: [
-				...productCatalogDescription.TierThree.benefits,
-				{
-					copy: `Unlimited access to the Guardian's 200-year newspaper archive`,
-					isNew: true,
-					tooltip: `Look back on more than 200 years of world history with the Guardian newspaper archive. Get digital access to every front page, article and advertisement, as it was printed ${
-						countryGroupId === 'GBPCountries' ? '' : ' in the UK'
-					}, since 1821.`,
-				},
-			],
-		},
-		SupporterPlus: {
-			...productCatalogDescription.SupporterPlus,
-			benefits: [
-				...productCatalogDescription.SupporterPlus.benefits,
-				{
-					copy: 'Unlimited access to the Guardian Feast App',
-					isNew: true,
-					tooltip:
-						'Make a feast out of anything with the Guardian’s new recipe app. Feast has thousands of recipes including quick and budget-friendly weeknight dinners, and showstopping weekend dishes – plus smart app features to make mealtimes inspiring.',
-				},
-			],
-			offers: [],
-		},
-	};
-}
+export const productCatalogDescriptionNewBenefits: Record<
+	ProductKey,
+	ProductDescription
+> = {
+	...productCatalogDescription,
+	TierThree: {
+		...productCatalogDescription.TierThree,
+		benefits: [
+			...productCatalogDescription.TierThree.benefits,
+			{
+				copy: `Unlimited access to the Guardian's 200-year newspaper archive`,
+				isNew: true,
+				tooltip: `Look back on more than 200 years of world history with the Guardian archive. Get digital access to every front page, article and advertisement printed in the newspaper from 1981.`,
```

Comment:
Is `1981` here correct? Elsewhere we mention `1821`.

Reply:
Good question, it matches the Figma file change.  Does not seem to align the archive availability with  200yrs to be fair. Will check now.

Reply:
Was 1821, updated